tests/websockets/test_graphql_transport_ws.py

```diff
@@ -183,7 +183,8 @@ async def test_close_twice(
     ) as ws:
         transport_close = mocker.patch.object(ws, "close")
 
-        await ws.send_json(ConnectionInitMessage(payload=None).as_dict())
+        # TODO: payload is set to 1 to force close, but is this right?
+        await ws.send_json(ConnectionInitMessage(payload=1).as_dict())
```

Comment:
@bellini666 can you take a look at this? I think you added this test 😊

Reply:
Yeah, it was in [this PR](https://github.com/strawberry-graphql/strawberry/pull/2341), which had more changes other than the test itself but got superseded by another fix and we decided to keep just the tests.

This seems to be fine, what we want from this test is to send a wrong payload to `ConnectionInitMessage`, which would cause it to be closed immediatly, and then check that the init timeout handler doesn't try to close it again.

Passing `1` or any invalid payload is fine! :). Maybe just make it more specific that `1` is an arbitrary invalid value, because otherwise it could make someone think that `1` means close =P